code/modules/mining/mine_items.dm

```diff
@@ -35,32 +34,146 @@
 	icon_state = "lantern"
 	desc = "A mining lantern."
 	brightness_on = 4			// luminosity when on
-	light_power = 0.75
+	light_power = 1
+	light_range = 6
```

Comment:
`light_range` will be overridden by `brightness_on` when the lantern is turned on.